internal/service/mq/broker_test.go

```diff
@@ -1119,6 +1119,7 @@ func TestAccMQBroker_Update_securityGroup(t *testing.T) {
 	})
 }
 
+/*
```

Comment:
Can this be removed entirely? Or will it be re-enabled at some point in the future?

Reply:
I'm going to leave it in as it would be nice to test the functionality if the versions line up in the future.